config/main.py

```diff
@@ -3252,6 +3252,10 @@ def reload(ctx, no_dynamic_buffer, no_delay, dry_run, json_data, ports, verbose)
     if buffer_model_updated:
         print("Buffer calculation model updated, restarting swss is required to take effect")
 
+    if not status:
+        click.echo("Some keys in APPL_DB didn't update successfully, please reload configuration.")
```

Comment:
I'm not sure whether we still need message here since we already printed here https://github.com/jianyuewu/sonic-utilities/pull/4/files#diff-8bb4eed8e9e1f9974cbb7b46799411ecc318bae391f48735e7ad84a8122809a0R797
@keboliu  what do you think?

Reply:
Agree, there is print previously, should be already informative, and latter has sys.exit(1), should be not needed.